config/locales/ca_admin.yml

```diff
@@ -3,3 +3,12 @@ ca:
     attributes:
       attachment_collection:
         weight: Pes
+  decidim:
```

Comment:
do we need to change this translations only to split the multiselect in two?

Reply:
Oops. Should I make another PR?

Reply:
Yes do it please.

Reply:
Done: #90